<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OnPointLink </string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#login, #MainWindow
{
    /*背景色，蓝到淡黄渐变*/
    /*渐变色:rgba中最后一值为透明度,0_255&lt;-&gt;完全透明_不透明*/
    background-color: qradialgradient(cx:0.25, cy:0.25, radius:0.75, fx:0.618 fy:0.618,
                                        stop:0 rgba(20,150,220,240),
                                        stop:1 rgba(255,245,220,240));
}

#login
{
    /*宽高*/
    width: 800px;
    height: 500px;
}

#label_picture
{
    /*背景图片，注意格式*/
    border-image:url(:/res/background-images/login1.jpg) 4 4 4 4 strech strech;
    /*为logo采用特殊字体*/
    font-family: Chiller;
    font-size: 80px;
    font-weight:bold;
}


#pushButton_Login
{
    font-family:STXinwei;
    font-size: 24px;
    font-weight:bold;
}


QComboBox
{
    /*背景，淡黄*/
    background: rgb(255,245,220);
    /*边框，蓝绿*/
    border-color: rgb(0,140,180);
    border-style: groove;
    border-radius: 3px;
    border-width: 2px;
    /*前景/字体*/
    color: black;
    font-family: consolas;
    font-size: 24px;
    /*内边距*/
    padding-top: 2px;
    padding-left: 2px;
    /*宽高*/
    height: 30px;
}
/*下拉框格式*/
QComboBox QAbstractItemView
{
    /*背景，淡黄*/
    background: rgb(255,245,220);
    /*前景/字体*/
    color: black;
    font-family: consolas;
    font-size: 24px;
    /*内边距*/
    padding-top: 2px;
    padding-left: 2px;
    /*宽高*/
    height: 30px;
}
/*选中效果*/
QComboBox:on
{
    padding-top: 3px;
    padding-left: 4px;
}
/*下拉箭头容器*/
QComboBox::drop-down
{
    width: 32px;
    height: 32px;
}

/*下拉箭头*/
QComboBox::down-arrow
{
    image: url(:/res/icons/xia_la.png);
}
/*下拉箭头选中效果*/
QComboBox::down-arrow:on
{
    image: url(:/res/icons/shang_la.png);
}

/*确认框*/
QCheckBox
{
    /*前景/字体*/
    color: black;
    font-family: consolas;
    font-size: 24px;
    /*内边距*/
    padding-left:5px;
}

QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgba(255, 255, 255, 0);
    border: 0px solid #b1b1b1;
        width: 30px;
        height: 30px;
}

QCheckBox::indicator:unchecked
{
    image: url(:/res/icons/kaiguan_guan.png);
}

QCheckBox::indicator:checked
{
    image: url(:/res/icons/kaiguan_kai.png);
}


/*按钮*/
QPushButton,QRadioButton
{
    /*背景，偏深蓝*/
    background-color: rgb(0,180,240);
    /*边框,淡黄*/
    border-color: rgba(240,230,210,220);
    border-radius: 5px;
    border-style: groove;
    border-width: 2px;
    /*前景/字体，淡蓝*/
    color: rgb(240,255,255);
    font-family: consolas;
    font-size: 20px;
    font-weight:bold;
    /*内边距*/
    /*宽高*/
    height:30px;
}

/*悬停时，背景变为淡蓝色，字体变为白色*/
QRadioButton:hover,QPushButton:hover
{
    background-color: rgb(100,180,250);
    color: white;
}

/*按下时，背景变为蓝绿色，显示边框效果*/
QPushButton:pressed,QRadioButton:pressed
{
    /*背景*/
    background-color: rgb(0,140,180);
    /*边框*/
    border-style: inset;
    /*前景/字体*/
    color: white;
    /*内边距*/
    padding-left:2px;
    padding-top:2px;
}


QLabel
{
    font-family: consolas;
    font-size: 24px;
    font-weight:bold;
    /*设置QLabel字体为居中*/
    qproperty-alignment:AlignCenter;
}


QTableWidget
{
    /*背景，淡黄*/
    background: rgb(255,245,220);
    /*选中背景，蓝色*/
    selection-background-color: rgb(10, 140, 220);
    /*边框，蓝绿*/
    border-color: rgb(0,140,180);
    border-style: groove;
    border-radius: 3px;
    border-width: 2px;
    /*前景/字体*/
    color: black;
    font-family: consolas;
    font-size: 24px;
    /*宽高*/
    height: 30px;
}

/* 以下加入 */

QPlainTextEdit, QListWidget, QLineEdit, QTreeWidget
{
    /*背景，淡黄*/
    background: rgb(255,245,220);
    /*选中背景，蓝色*/
    selection-background-color: rgb(10, 140, 220);
    selection-color: white;
    /*无边框*/

    /*前景/字体*/
    color: black;
    font-family: consolas;
    font-size: 16px;        
}

QTreeWidget::item
{
    /*背景，淡黄*/
    background: rgb(255,245,220);
    /*选中背景，蓝色*/
    selection-background-color: rgb(10, 140, 220);
    selection-color: white;
    /*无边框*/

    /*前景/字体*/
    color: black;
    font-family: consolas;
    font-size: 16px;        
}

QPlainTextEdit, QLineEdit, QTreeWidget 
{
    /*窄边框，灰色*/
    border-color: rgb(160, 160, 160);
    border-style: solid;
    border-radius: 2px;
    border-width: 1px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="sideFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>consolas</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>OP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="picture">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>45</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_message">
        <property name="text">
         <string>M</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_friends">
        <property name="text">
         <string>F</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_settings">
        <property name="text">
         <string>S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>N</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_middle">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_mid_0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidget_message">
         <property name="font">
          <font>
           <family>consolas</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_mid_1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_searchUser"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_addFriend">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">padding : 5px 10px</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_refresh">
           <property name="styleSheet">
            <string notr="true">padding : 5px 10px</string>
           </property>
           <property name="text">
            <string>Ref</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_friendRequest">
         <property name="text">
          <string>Friend request</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_createGroup">
         <property name="text">
          <string>Create group</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget_friend">
         <column>
          <property name="text">
           <string notr="true">分组</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_mid_2">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_main">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_main_0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="textEdit_show">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_choDoc">
            <property name="text">
             <string>选择文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_input">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_main_1">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_userInfo_OID">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_userInfo"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_blockfriend">
           <property name="text">
            <string>Block</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_deletefriend">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_changegroup">
           <property name="text">
            <string>Change group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_addThisUser">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_main_2">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
